lib/GenomeFileUtil/core/FastaGFFToGenome.py

```diff
@@ -554,8 +437,8 @@ def _update_phytozome_features(self,feature_list):
                         break
                 if(old_id is None):
                     #This should be an error
-                    print ("Cannot find unique ID, PACid, or pacid in GFF attributes: ",\
-                               feature_list[contig][i][contig],feature_list[contig][i][source],feature_list[contig][i][attributes])
+                    print ("Cannot find unique ID, PACid, or pacid in GFF "
```

Comment:
The nature of the GFF final column with free-form key value pairs brings up the potential need for maybe another map on each feature (Feature, CDS, mRNA, Non-coding feature) called additional attributes. Which would keep the key value pairs that we could not make sense of, thus no information is lost. From a KBase standpoint we could really only use that information for downloads and feature level visualizations. Does not need to be addressed now, and can potentially cause some size issues, but I think it is worth discussing.